addons/l10n_ar/models/account_move.py

```diff
@@ -360,11 +361,62 @@ def _l10n_ar_get_invoice_totals_for_report(self):
             for subtotal in tax_totals['subtotals']
             for tax_group in subtotal['tax_groups']
         }
-        tax_group_ids_to_exclude = self.env['account.tax.group'].browse(tax_group_ids).filtered('l10n_ar_vat_afip_code').ids
+        tax_group_ids_to_exclude = self.env['account.tax.group']\
+            .browse(tax_group_ids)\
+            .filtered(lambda tax_group: (
+                not self._l10n_ar_is_tax_group_other_national_ind_tax(tax_group)
+                and not self._l10n_ar_is_tax_group_vat(tax_group)
+            )).ids
         if tax_group_ids_to_exclude:
-            return self.env['account.tax']._exclude_tax_groups_from_tax_totals_summary(tax_totals, tax_group_ids_to_exclude)
+            tax_totals = self.env['account.tax']._exclude_tax_groups_from_tax_totals_summary(tax_totals, tax_group_ids_to_exclude)
         return tax_totals
 
+    def _l10n_ar_get_invoice_custom_tax_summary_for_report(self):
+        """ Get a new tax details for RG 5614/2024 to show ARCA VAT and Other National Internal Taxes. """
+        if self.l10n_latam_document_type_id.code not in ('6', '7', '8'):
```

Comment:
@zaoral We still render that part even if `_l10n_ar_include_vat` returns `False`. Is it intended?

Reply:
Yes, before this new RG we do not want any VAT tax in the include_vat documents (final consumer documents - letter B documents)

With this change, we need to show them, but in a particular way: not detailed information, only totals by tax type. We also need to show the national/internal taxes in a separate table. They need to be added at the end of the report with a special caption.